README.md

```diff
@@ -79,6 +106,8 @@ The engine is configured by the following ENV variables:
 You might want to use [dotenv-rails](https://github.com/bkeepers/dotenv)
 to make ENV variables management easy.
 
+**When using a multi application setup, it's best to keep BOOKINGSYNC_APP_ID and BOOKINGSYNC_APP_SECRET blank**
```

Comment:
Why? what will happen if we don't do that?

Reply:
Nothing, yet that will ensure that it can't work in single app mode which would behave differently than expected. Having this nullify is just an extra prevention.

Reply:
How about mentioning it? That it won't have any impact on anything?